pkg/views/build/list/view.go

```diff
@@ -29,10 +25,6 @@ type RowData struct {
 func ListBuilds(buildList []apiclient.Build, apiServerConfig *apiclient.ServerConfig) {
 	SortBuilds(&buildList)
 
-	re := lipgloss.NewRenderer(os.Stdout)
-
-	headers := []string{"ID", "State", "Prebuild ID", "Created", "Updated"}
-
 	data := [][]string{}
 
 	for _, pc := range buildList {
```

Comment:
From what I can see, we can reduce this loop to 
`data = append(data, getRowFromData(pc))` 
(Seems to me that `getTableRowData` is an unnecessary extra step)

P.S. I believe this to be the case in the other files as well.